examples/flexvolume/nginx.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    volumeMounts:
+    - name: test
+      mountPath: /data
+    ports:
+    - containerPort: 80
+  volumes:
+  - name: test
+    flexVolume:
+      driver: "kubernetes.io/lvm"
+      fsType: "ext4"
+      options:
+        volumeID: "vol1"
+        size: "1000m"
+        volumegroup: "kube_vg"
+        fsType: "ext4"
```

Comment:
On second thought, shall we return a status in "FlexVolumeDriverStatus", "mounted=true". That way we can limit the callout to one call during setup & teardown? 
With multiple calls for filesystems like CIFS/NFS, attach and detach should not do anything, which will be confusing. Let me know what you think.